app/server/api/example/[id].patch.ts

```diff
@@ -1,16 +1,15 @@
 import { defineEventHandler } from 'h3'
 import type { CompatibilityEvent } from 'h3'
-import { z } from 'zod'
+import { parseBodyAs, parseDataAs, parseParamsAs, z } from '@sidestream-tech/nuxt-sidebase-parse'
 import { Example, exampleFull, exampleUpdate } from '~/server/database/entities/Example'
-import { parseBodyAs, parseDataAs, parseParamsAs } from '~/server/helpers'
 
 const paramsSchema = z.object({
   id: z.string().uuid(),
 })
 
 export default defineEventHandler(async (event: CompatibilityEvent) => {
   // Parse the request parameters (so the dynamic data that is part of the URL, e.g.: `/example/1` where `1` is the id)
-  const params = await parseParamsAs(event, paramsSchema)
+  const params = parseParamsAs(event, paramsSchema)
```

Comment:
Why do we not need to await it anymore?

Reply:
Because getting the parameters from the event is not actually an async operation - only parsing the body is.

In the old implementation of the helpers, everything was generated and then it was easier to make it async. Not everything is explicitly written out and so we can differentiate.

See https://github.com/sidestream-tech/sidebase-libs/blob/main/packages/nuxt-sidebase-parse/src/index.ts for the details